src/api/db/migrate/20160317164117_sanitize_invalid_repository_names.rb

```diff
@@ -0,0 +1,17 @@
+class SanitizeInvalidRepositoryNames < ActiveRecord::Migration
+  def up
+    Repository.where("name REGEXP ?", "^_|[:\/]+").find_each do |repository|
+      repository.name = repository.name.gsub(/^_|[:\/]+/, "")
+      repository.name = "please_enter_a_valid_name_#{SecureRandom.hex(5)}" if repository.blank?
+      repository.save!
```

Comment:
Huh, okay. I was hoping this would cover all development and productive instances of OBS.